components/interactive_dialog/interactive_dialog.jsx

```diff
@@ -51,18 +53,20 @@ export default class InteractiveDialog extends React.Component {
         const {elements} = this.props;
         const values = this.state.values;
         const errors = {};
-        elements.forEach((elem) => {
-            const error = checkDialogElementForError(elem, values[elem.name]);
-            if (error) {
-                errors[elem.name] = (
-                    <FormattedMessage
-                        id={error.id}
-                        defaultMessage={error.defaultMessage}
-                        values={error.values}
-                    />
-                );
-            }
-        });
+        if (elements != null) {
```

Comment:
Suggest: `if (elements) {` 

which would help protect against problems with undefined, null, and type coercion